src/MonthPicker.js

```diff
@@ -35,11 +35,12 @@ along with this program.  If not, see
     // conflict with Bootstrap.js button (#35)
     $.widget.bridge('jqueryUIButton', $.ui.button);
 
+    var _isNewApi = $.ui.version >= '1.12';
```

Comment:
The tests are paling because this tests is incorrect.

The unit tests are using version 1.9.2 of jQuery UI and this test returns `true` because the number `1.9` is greater than `1.12`.

I pushed a fix for this to my [jquery_1.12_support branch](https://github.com/benjamin-albert/jquery-ui-month-picker/tree/jquery_1.12_support).